fabric-rendering-v0/src/main/resources/fabric-rendering-v0.mixins.json

```diff
@@ -3,10 +3,6 @@
   "package": "net.fabricmc.fabric.mixin.client.rendering",
   "compatibilityLevel": "JAVA_8",
   "client": [
-    "MixinBlockColorMap",
-    "MixinBlockEntityRenderManager",
-    "MixinEntityRenderManager",
-    "MixinItemColorMap",
     "MixinWorldRenderer"
```

Comment:
Why is this left?

Reply:
It is to call the old InvalidateRenderStateCallback. I might just remove this and maybe create an entrypoint to register a listener to the new one that will call the old ones? I don't really know how I would do this.

Reply:
I believe it would be better to register a handler for the v1 module's event that invokes the v0 event.